nautobot_circuit_maintenance/migrations/0014_delete_note.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.16 on 2024-09-13 08:49
+
+from django.db import migrations
+
+
+def move_data_to_new_model(apps, schema_editor):
+    CircuitMaintenance = apps.get_model('nautobot_circuit_maintenance', 'CircuitMaintenance')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    OldModel = apps.get_model('nautobot_circuit_maintenance', 'Note')
+    NewModel = apps.get_model('extras', 'Note')
+
+    for old_obj in OldModel.objects.all():
+        NewModel.objects.create(
+            note=old_obj.title + "\n" + old_obj.comment,
+            assigned_object_type=ContentType.objects.get_for_model(CircuitMaintenance),
+            assigned_object_id=old_obj.maintenance.id,
+            user_name="migration"
+        )
+
+def reverse_move_data(apps, schema_editor):
+    OldModel = apps.get_model('nautobot_circuit_maintenance', 'Note')
+    NewModel = apps.get_model('extras', 'Note')
+
+    for new_obj in NewModel.objects.all():
+        OldModel.objects.create(
+            title=new_obj.note[:20], # truncate to 20
+            content=new_obj.note,
+            maintenance=new_obj.assigned_object
+        )
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('nautobot_circuit_maintenance', '0013_rename_site_search_job'),
+    ]
+
+    operations = [
+        migrations.RunPython(move_data_to_new_model, reverse_move_data),
+        migrations.DeleteModel(
+            name='Note',
+        ),
+    ]
```

Comment:
This needs to be split up into separate migration files. From https://docs.djangoproject.com/en/4.2/ref/migration-operations/#runpython

> On databases that do support DDL transactions (SQLite and PostgreSQL), RunPython operations do not have any transactions automatically added besides the transactions created for each migration. Thus, on PostgreSQL, for example, you should avoid combining schema changes and RunPython operations in the same migration or you may hit errors like OperationalError: cannot ALTER TABLE "mytable" because it has pending trigger events.